VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -171,4 +166,10 @@ protected void hideMessageInSitu(){
         }
     }
 
+    private void showErrorDialog(String message) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setMessage(message)
+                .setPositiveButton(R.string.label_ok, null)
+                .create().show();
```

Comment:
Should we add the 'Try again' button? It's suggested to do so in the wiki, but it would increase the scope of the implementation. I guess we could create another ticket for implementing that after merging this pull request.